tests/xharness/Jenkins/TestData.cs

```diff
@@ -9,7 +9,7 @@ class TestData
 		public string Variation;
 		public string MTouchExtraArgs;
 		public string MonoBundlingExtraArgs; // mmp
-		public string KnownFailure;
+		public (string HumanMessage, string IssueLink)? KnownFailure;
```

Comment:
How does `?` work here?

Reply:
Named tuples are value types, not reference types, we add the ? to allow the value to be null, same as with an int.

Reply:
c#8 nullability, it means it can be null (top of file has `#nullable enable`)

Reply:
@spouliot not this one ;) 

But I should add it.

Reply:
@whitneyschmidt for ref, a named tuple is under the hood a https://docs.microsoft.com/en-us/dotnet/api/system.valuetuple?view=netcore-3.1 that is a struct, structs can't be null, so I added the ? to do a nullable one.

Reply:
when you think too much about something then you see it everywhere ;-)